README.md

```diff
@@ -92,53 +122,62 @@ resource anymore, it deletes all the previously created resources.
+To create a Docker image instead, execute `make build.docker`. You can then push your Docker image to the Docker 
+registry of your choice.
 
 ## Deploy
 
-To deploy the ingress controller, use [this yaml](deploy/ingress-controller.yaml) as the descriptor. You can customize the image used in the yaml. 
-
-We provide `registry.opensource.zalan.do/teapot/kube-aws-ingress-controller:latest` as a publicly usable Docker image built from this codebase. You can deploy it by executing the following command, after replacing the placeholder for the region:
+To deploy the ingress controller, use the [example YAML](deploy/ingress-controller.yaml) as the descriptor. You can 
+customize the image used in the example YAML file. 
 
-```
-kubectl apply -f deploy/ingress-controller.yaml
-```
+We provide `registry.opensource.zalan.do/teapot/kube-aws-ingress-controller:latest` as a publicly usable Docker image 
+built from this codebase. You can deploy it with 2 easy steps:
+- Replace the placeholder for your region inside the example YAML, for ex., `eu-west-1`
+- Use kubectl to execute the command  `kubectl apply -f deploy/ingress-controller.yaml`
 
 ## Trying it out
 
-The Ingress Controller's responsibility is limited to managing load balancers, as described above. To have a fully functional setup, additionally to the ingress controller, you can use [Skipper](https://github.com/zalando/skipper) to route the traffic to the application. The setup follows what's described [here](https://kubernetes-on-aws.readthedocs.io/en/latest/user-guide/ingress.html).
+The Ingress Controller's responsibility is limited to managing load balancers, as described above. To have a fully 
+functional setup, additionally to the ingress controller, you can use [Skipper](https://github.com/zalando/skipper) 
+to route the traffic to the application. The setup follows what's 
+described [here](https://kubernetes-on-aws.readthedocs.io/en/latest/user-guide/ingress.html).
 
-You can deploy `skipper` as a `DaemonSet` using [this yaml](deploy/skipper.yaml), and by executing the following commands:
+You can deploy `skipper` as a `DaemonSet` using [another example YAML](deploy/skipper.yaml) by executing the following command:
 
 ```
 kubectl apply -f deploy/skipper.yaml
 ```
 
-To complete the setup, you'll need to fulfill some additional requirements regarding security groups and IAM roles; [more info here](deploy/requirements.md).
+To complete the setup, you'll need to fulfill some additional requirements regarding security groups and IAM 
+roles; [more info here](deploy/requirements.md).
 
 ### DNS
 
-To have convenient DNS names for your application, you can use [External DNS](https://github.com/kubernetes-incubator/external-dns). It's not strictly necessary for this Ingress Controller to work, though.
+To have convenient DNS names for your application, you can use [mate](https://github.com/zalando-incubator/mate). 
```

Comment:
Is the README really up to date? this looks like a previous version of it (see the reference to `mate`).